packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -199,6 +207,41 @@ const HelperText = createComponentSlot(FormHelperText)<FieldContainerClassKey>({
     `,
 );
 
+const SecondaryHelperText = createComponentSlot(FormHelperText)<FieldContainerClassKey>({
+    componentName: "FormFieldContainer",
+    slotName: "secondaryHelperText",
+})(
+    ({ theme }) => css`
+        color: ${theme.palette.grey[300]};
+        margin-left: auto;
+
+        &.Mui-disabled {
+            color: ${theme.palette.grey[300]};
+        }
+    `,
+);
+
+const HelperTextWrapper = createComponentSlot("div")<FieldContainerClassKey>({
+    componentName: "FormFieldContainer",
+    slotName: "helperTextWrapper",
+})(
+    ({ theme }) => css`
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        gap: ${theme.spacing(3)};
+
+        svg {
```

Comment:
Please target the root-element of the `MuiIcon` component here, instead of the `svg` tag. 
Any mayby only the direct child, as (theoretically) there could be nested, custom-styled icons in the passed in node. 

```suggestion
        > .${svgIconClasses.root} {
```